eclipse-theia/theia
# vsx-registry: sanitize readme

#### What it does


The following pull-request sanitizes the rendered `readme` of extensions for security purposes. The sanitization is similar to other
areas of the code which make use of `dangerouslySetInnerHTML`.

#### How to test


1. start the application
2. open the extensions-view (<kbd>F1</kbd> > `Toggle Extensions View`)
3. verify the `readme` of the following (when double-clicking their entries in the tree):
  3.1 builtin extensions
  3.2 installed extensions
  3.3 extension search results

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
